fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/FluentValidator.java

```diff
@@ -527,6 +540,67 @@ public FluentValidator doValidate(ValidateCallback cb) {
         return this;
     }
 
+    
+    public FluentValidator doCustomValidate(ValidateCallback cb) {
```

Comment:
Copying the whole method but only left on parameter difference, this is not acceptable because it breaks common DRY(do not repeat yourself) rule. So could you refine it a better a way? maybe we can leverage varargs?

Also one more question, what is your purpose of specifying a error message to "on" method? I do not see the usage in any test cases. Could you provide more information? And maybe we can come up a better solution.

For you PR, I suggest you use it on your local maven repo first. Once we settle down, I will publish it to central repo asap.